src/contents/ui/main.qml

```diff
@@ -20,6 +20,24 @@ PlasmoidItem {
     readonly property bool textScrollingEnabled: plasmoid.configuration.textScrollingEnabled
     readonly property bool textScrollingResetOnPause: plasmoid.configuration.textScrollingResetOnPause
     readonly property int volumeStep: plasmoid.configuration.volumeStep
+    readonly property bool colorsFromAlbumCover: plasmoid.configuration.colorsFromAlbumCover
+    property color imageColor: Kirigami.Theme.textColor
+    property bool imageReady: false
+    property string backgroundColor: imageReady && colorsFromAlbumCover ? Kirigami.ColorUtils.tintWithAlpha(imageColor, "#000000", 0.5) : "transparent"
+    property string foregroundColor: imageReady && colorsFromAlbumCover ? Kirigami.ColorUtils.tintWithAlpha(imageColor, contrastColor, .6) : Kirigami.Theme.textColor
+    property string contrastColor: Kirigami.ColorUtils.brightnessForColor(backgroundColor) === Kirigami.ColorUtils.Dark ? "#ffffff" : "#000000"
```

Comment:
I would move these vars inside the compactRepresentation Item because they concern only the panel representation

Reply:
Done